lib/chewy.rb

```diff
@@ -48,7 +48,7 @@ def try_require(path)
 require 'chewy/fields/base'
 require 'chewy/fields/root'
 require 'chewy/journal'
-require 'chewy/railtie' if defined?(::Rails::Railtie)
+require 'chewy/railtie' if defined?(Rails::Railtie)
```

Comment:
What has changed we can skip the top module selector?

Reply:
This is what cop developer says 

> How Ruby searches constant is a bit complicated, and it can often be difficult to understand from the code whether the ‘::` is intended or not. Where `Module.nesting` is empty, there is no need to prepend `::`, so it would be nice to consistently avoid such meaningless `::` prefix to avoid confusion.

I've run all our specs on platform, and haven't seen any issues, so I suspect it's fine.

Reply:
Now I'm thinking maybe it makes sense to keep it, we cannot know why it has been added and considered gem is public, I'll bring it back

Reply:
Ok, I'm changing my opinion once again after careful inspection. In the same file for the previous version we `Rspec` and later `::Rspec`, so it seems there wasn't an intention to always use top-level consants.